packages/launcher/src/widget.tsx

```diff
@@ -298,7 +299,11 @@ function Card(
       <div className="jp-LauncherCard-icon">
         {kernel ? (
           item.kernelIconUrl ? (
-            <img src={item.kernelIconUrl} className="jp-Launcher-kernelIcon" />
+            <img
+              src={item.kernelIconUrl}
+              className="jp-Launcher-kernelIcon"
+              alt={trans.__('Kernel Icon')}
```

Comment:
Same: what is the rationale for not setting `alt=""`?

Reply:
The reason alt is not an empty string is because we believe it is not a decorative element and adds value to the end users.

Reply:
In the accessibility call today, we discussed a scenario in which the text description that a kernel provides to describe itself might make it unclear in the launcher that it is a kernel. Imagine a kernel called Coconut that labels itself as just "Coconut." In that case, it might be helpful to a screen reader to announce the launcher card as "Kernel Icon coconut".

We also discussed some alternative in the call. What if the alt text were something like "Create new notebook using kernel", so then the full accessible text for a Python 3 ipykernal notebook launcher card would read like so: "Create new notebook using kernel Python 3 (ipykernel)". That seems a bit verbose to me but clear.

My chief concern in asking these questions is to avoid a common pitfall, which is: in our desire to make things better for disabled users, we actually make them worse. I want to try to avoid that.

Reply:
Hi @gabalafou your comment makes sense and I completely understand where you are coming from. Thanks for the feedback. I have made the change that you have recommended for the alt text being "Create new notebook using kernel".